packages/forms-web-app/src/dynamic-forms/validator/booleanValidator.js

```diff
@@ -0,0 +1,10 @@
+const { body } = require('express-validator');
+const BaseValidator = require('./baseValidator.js');
+
+class BooleaValidator extends BaseValidator {
```

Comment:
Looks like there's a typo here - we're missing the `n` on `Boolean`.

Reply:
This has been renamed anyway